processor/src/chiplets/hasher/tests.rs

```diff
@@ -12,7 +12,7 @@ use vm_core::{
         MR_UPDATE_OLD_LABEL, NUM_ROUNDS, NUM_SELECTORS, RETURN_HASH_LABEL, RETURN_STATE_LABEL,
         STATE_COL_RANGE,
     },
-    code_blocks::CodeBlock,
+    code_blocks::CodeBlock, 
```

Comment:
Please remove this change.

Reply:
Will do.  `cargo fmt` is currently failing.  I will get this cleaned up on the next push.